micromasters/utils.py

```diff
@@ -19,6 +19,14 @@ def webpack_dev_server_url(request):
     return 'http://{}:{}'.format(webpack_dev_server_host(request), settings.WEBPACK_DEV_SERVER_PORT)
 
 
+def remove_key(dictionary, key):
+    """
+    Helper method to remove a key from a dict and return the dict
+    """
+    del dictionary[key]
+    return dictionary
```

Comment:
what's the benefit of doing this instead of just calling `del dictionary[key]` wherever you need it?